configure_macports

```diff
@@ -101,9 +101,10 @@ main()
     esac
 
     if [ -f "${parameterfile}" -a -r "${parameterfile}" ]; then
-	macports_version=$(yq ".version // \"${macports_version}\"" < "${parameterfile}")
-	macports_prefix=$(yq ".prefix // \"${macports_prefix}\"" < "${parameterfile}")
-    fi
+		macports_version=$(yq ".version // \"${macports_version}\"" < "${parameterfile}")
+		macports_prefix=$(yq ".prefix // \"${macports_prefix}\"" < "${parameterfile}")
+		. "${subrdir}/check_version.sh" "${macports_version}"
```

Comment:
Sourcing files that have an effect beyond defining yields very confusing programs. I would recommend to move your logic to the `configure_macports` file and add the configuration details to the output of the action. For the logic maybe

leave if the `if [ -f "${parameterfile}" ]` untouched and add another block

```
if [ "${macports_version}" = 'LATEST' ]; then
  macports_version="$(determine_latest_macports_version)"
fi
```

where you define the `determine_latest_macports_version` with your curl/awk combo.